scripts/install-control-cluster.sh

```diff
@@ -204,18 +191,27 @@ kubernetes-dashboard:
   rbac:
     clusterReadOnlyRole: true"
 
+set -x
+
+# Update Proxy if required
+if check_proxy_vars; then
+   echo "Setting Proxy"
+   export http_proxy=$HTTP_PROXY
+   export https_proxy=$HTTPS_PROXY
+   export no_proxy=$api_server,$pod_cidr,$NO_PROXY
+fi
+
```

Comment:
setting -x in L194 still leads to the echo command being unnecessarily printed here and other places below too, for example:

```
+ check_proxy_vars
+ '[' -n http://proxy.esl.cisco.com:80 ']'
+ return 0
+ echo 'Setting Proxy'
Setting Proxy
```